### DISCUSSION TITLE
Thread for discussions on Training GGML models like Falcon 40b

### DISCUSSION THREAD

**Discussion by linuxmagic-mp** [cmp-nct/ggllm.cpp#45] [Ideas]
Just opening a discussion idea for Training GGML models.  Now that it is fairly easy and straight forward to make GGML versions of models such as Falcon 40b, more and more questions all over the place on 'Training'.  If you have experience doing so, and can share your tips, tricks and successes (or failures) here would be a good place to post them.  As well, if you have any questions on Training, feel free to post them here as well.  NOTE: This is NOT about fine-tuning, but raw training

**Comment by linuxmagic-mp** [comment]
A small 'note' in the readme regarding the latest state of training GGML models might also be helpful, just searching for a good link to suggest.. recommendations?

**Reply by cmp-nct** [reply]
I think a section on training and one on finetuning would be well placed in the Wiki
Though I'd like to get training and fine tuning on ggllm.cpp natively working
